ext/storage_sources/azure_store/azure_log_system.cpp

```diff
@@ -0,0 +1,89 @@
+/*-
+ * Public Domain 2014-present MongoDB, Inc.
+ * Public Domain 2008-2014 WiredTiger, Inc.
+ *
+ * This is free and unencumbered software released into the public domain.
+ *
+ * Anyone is free to copy, modify, publish, use, compile, sell, or
+ * distribute this software, either in source code form or as a compiled
+ * binary, for any purpose, commercial or non-commercial, and by any
+ * means.
+ *
+ * In jurisdictions that recognize copyright laws, the author or authors
+ * of this software dedicate any and all copyright interest in the
+ * software to the public domain. We make this dedication for the benefit
+ * of the public at large and to the detriment of our heirs and
+ * successors. We intend this dedication to be an overt act of
+ * relinquishment in perpetuity of all present and future rights to this
+ * software under copyright law.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND,
+ * EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF
+ * MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT.
+ * IN NO EVENT SHALL THE AUTHORS BE LIABLE FOR ANY CLAIM, DAMAGES OR
+ * OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE,
+ * ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR
+ * OTHER DEALINGS IN THE SOFTWARE.
+ */
+
+#include "azure_log_system.h"
+
+// Constructor for azure_log_system that calls to set the WiredTiger verbosity level.
+azure_log_system::azure_log_system(WT_EXTENSION_API *wt_api, uint32_t wt_verbosity_level)
+    : _wt_api(wt_api)
+{
+    set_wt_verbosity_level(wt_verbosity_level);
+}
+
+// Find Azure Logger level given WiredTiger verbosity level returns Error if not found.
+const Azure::Core::Diagnostics::Logger::Level
+wt_to_azure_verbosity_level(int32_t wt_verbosity_level)
```

Comment:
The types for `wt_verbosity_level` should be unified between here and the constructor - currently one's a `uint32_t` and the other is an `int32_t`.

Reply:
Thanks for picking this up! I've unified them now.